projects/storefrontapp-e2e-cypress/cypress/integration/regression/cart/cart.core-e2e-spec.ts

```diff
@@ -75,6 +77,9 @@ describe('Cart', () => {
             'spartacus⚿electronics-spa⚿cart',
             JSON.stringify(storage)
           );
+
+          cy.wait(2000);
```

Comment:
why here need wait 2s?

Reply:
reason for that, it flakes sometimes and not having a not found cart

the reason I can say is when cypress does this action by changing local storage, it usually just visits the cart part right away (very fast).

With this added static guard after setting the local storage, we make it wait to hopefully stabilize before we visit the cart page

Reply:
You can find it working now from the last 8 runs https://dashboard.cypress.io/projects/k3nmep/runs?branches=%5B%7B%22label%22%3A%22feature%2FCXSPA-1587%22%2C%22suggested%22%3Afalse%2C%22value%22%3A%22feature%2FCXSPA-1587%22%7D%5D&committers=%5B%5D&flaky=%5B%5D&page=1&status=%5B%5D&tags=%5B%5D&timeRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D

But all the runs you see from the dashboard that is filtered to this branch shows fixes according to the commit